test/Autofac.Test/Features/Decorators/DecoratorTests.cs

```diff
@@ -128,10 +128,10 @@ public void DecoratedInstancePerLifetimeScopeRegistrationCanIncludeOtherServices
             var decoratedServiceInstance = container.Resolve<IDecoratedService>();
             Assert.IsType<DecoratorA>(decoratedServiceInstance);
 
-            Assert.Same(serviceInstance, decoratedServiceInstance);
+            Assert.Same(serviceInstance, decoratedServiceInstance.Decorated);
```

Comment:
Man, I did a horrible job at getting these tests in here. :( Thanks for fixing them.